include/glow/Partitioner/Partitioner.h

```diff
@@ -83,10 +82,12 @@ class Partitioner {
                                      llvm::StringRef backendName);
 
   /// Duplicates all networks in the module order to saturate the Host.
-  void saturateHost(unsigned logicalDeviceCount);
+  void saturateHost(unsigned logicalDeviceCount, const DAGListTy &partitions);
 
-  FunctionToBackendNameMap
-  backendBasedPartition(Function *F, std::vector<Backend *> &backends,
+  /// Do the partition based on backend types.
```

Comment:
could you add some description what F is and what's stored in functobackend map etc.